src/helpers/figures/charts/chart_common.ts

```diff
@@ -273,6 +279,49 @@ export function toExcelDataset(getters: CoreGetters, ds: DataSet): ExcelChartDat
     };
   }
 
+  let trend: ExcelChartTrendConfiguration | undefined = undefined;
+  if (ds?.trend?.type) {
+    trend = {
+      type: CHART_TRENDLINE_TYPE_CONVERSION_MAP_REVERSE[ds.trend.type],
+    };
+    if (ds.trend.color) {
+      trend = {
+        ...trend,
+        color: ds.trend.color,
+      };
```

Comment:
```suggestion
      trend.color = ds.trend.color;
```
are all the spread of this function really necessary ? There's no need to create a new object every time we want to set a property value

Reply:
We cannot do `trend.color = ds.trend.color` because the `color` property is readonly